examples/todo/package.json

```diff
@@ -14,7 +14,7 @@
     "express": "^4.13.3",
     "express-graphql": "^0.4.0",
     "graphql": "^0.4.7",
-    "graphql-relay": "^0.4.0",
+    "graphql-relay": "^0.3.3",
```

Comment:
Well there is no such package 0.4.0 or at least not in npm neither in github repo. The latest package is 0.3.3.